### PR TITLE
TM-3819, cycle search results

### PR DESCRIPTION
**Pull request by M00staff**
Might as well throw this up for a review while I'm out - feel free to go willy nilly and i'll address when i get back

[Ticket](https://metaphase.atlassian.net/browse/TM-3819)

image

### CODE REVIEW

File: src/Components/CycleManagement/CycleSearchCard.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    cycle_name,
+    cycle_status,
+    cycle_category,
+    cycle_begin_date,
+    cycle_end_date,
+    cycle_excl_position,
+    cycle_post_view,
+  } = props;
+
+  const cycleLink = `/profile/bureau/cyclemanagement/${id}`;
```
Comment (M00staff) [inline src/Components/CycleManagement/CycleSearchCard.jsx:18]: @pveland621 ^^ this is going to link to the page you're building out .. the naming and pattern is up to you, this is just a placeholder

**Comment by elizabeth-jimenez** [comment]
We'll want to set up pagination in this PR

**Comment by elizabeth-jimenez** [comment]
The designs dont have a left border - is that intentional?

**Comment by elizabeth-jimenez** [comment]
image
We don't want text to stack(CycleStart) before it absolutely has to

**Comment by elizabeth-jimenez** [comment]
You'll also want to add a shadow to your cards.

**Review by elizabeth-jimenez — CHANGES_REQUESTED** [review]
- Pagination
- Left border
- Stacking Text
- Shadow

**Comment by elizabeth-jimenez** [comment]
Additional changes:
- Icon discrepancy between menu and header.
- i think we are good to remove 'Search for an existing cycle' - i think that's a left over label from the search box
- honestly we might be ok to remove 'Cycle Search' too - gonna confirm with Tim

File: src/actions/cycleManagement.js [hunk 2]
```diff
@@ -1,7 +1,76 @@
...
+}
+const dummyDataToReturn = (query) => new Promise((resolve) => {
+  const { limit } = query;
+  resolve({
+    results: dummyData.slice(0, limit),
+    count: dummyData.length,
+    next: null,
+    previous: null,
+  });
+});
```
Comment (M00staff) [inline src/actions/cycleManagement.js:73]: I think everything above was overkill - but i wasn't sure how best to make sure the results per page for the pagination was working properly 🤷

need to update the mock and what not to be sure of the rest of the pagination - but I'm sure that is way out of scope for this ticket.

**Comment by cagbakwu** [comment]
> Might as well throw this up for a review while I'm out - feel free to go willy nilly and i'll address when i get back
> 
> [Ticket](https://metaphase.atlassian.net/browse/TM-3819)
> 
> image

Dark mode for the win!

**Review by cagbakwu — APPROVED** [review]
Besides the CSS variables to use, it looks good to me.

File: src/actions/cycleManagement.js [hunk 3]
```diff
@@ -30,27 +99,28 @@ export function cycleManagementFetchData(query = {}) {
       dispatch(cycleManagementFetchDataLoading(true));
       dispatch(cycleManagementFetchDataErrored(false));
     });
-    const q = convertQueryToString(query);
-    const endpoint = `sweet/new/endpoint/we/can/pass/a/query/to/?${q}`;
+    // const q = convertQueryToString(query);
+    // const endpoint = `sweet/new/endpoint/we/can/pass/a/query/to/?${q}`;
+    // api().get(endpoint)
```
Comment (M00staff) [inline src/actions/cycleManagement.js:104]: commented out until we get data from the mock

**Review by scott062 — APPROVED** [review]

**Pull request merged**